Makefile

```diff
@@ -0,0 +1,74 @@
+openapi-generator-version:=6.0.1
+openapi-generator-url:=https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/$(openapi-generator-version)/openapi-generator-cli-$(openapi-generator-version).jar
+openapi-generator-jar:=target/openapi-generator-cli.jar
+openapi-generator-cli:=java -jar $(openapi-generator-jar)
+
+generator:=java
+library:=okhttp-gson
+services:=payments
+models:=src/main/java/com/adyen/model
+output:=target/out
+
+
+# Generate models (for each service)
+models: $(services)
+
+binlookup: spec=BinLookupService-v52
```

Comment:
will this cause a big update even though we are trying to update the libraries one by one?